packages/linter/builders.json

```diff
@@ -20,6 +20,7 @@
     "eslint": {
       "implementation": "./src/executors/eslint/lint.impl",
       "schema": "./src/executors/eslint/schema.json",
+      "hasher": "./src/executors/lint/hasher",
```

Comment:
@vsavkin after much time spent digging we pinpointed our problems to this custom hasher.

The eslint hasher will often miss file-changes that the Nx core hasher doesn't. So in our project we quite often see issues where linting errors are introduced and Nx doesn't detect the changes, which leads to late-defects.
Removing the custom eslint hasher gives us much more predictable results.

Any thoughts?